content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
```

Comment:
After a quick read, I am wondering if this page belongs under `docs/reference/using-api/api-concepts`, or near the other API content pages,
https://5d579d7acb65dd0009e4cc90--kubernetes-io-vnext-staging.netlify.com/docs/reference/using-api/api-concepts/

Reply:
Frankly, I am not impressed with the fact that "using the Kubernetes API" and "accessing the API" are two different sections; it is not entirely obvious to me what is the rule the defines what goes in which section.  But the page being reviewed here is more like the authentication and authorization stuff in "accessing the API" (being about configured control over requests) than it is like the stuff in the "using" section.

Reply:
It feels to me like there's room for a short new page about this page in the reference section as well.
(one approach: get this merged to `dev-1.16` and then log an issue about adding the reference page).